File: SalesLibraries.FileManager/Business/Models/ExternalLibraryLinks/LibraryPage.cs
namespace SalesLibraries.FileManager.Business.Models.ExternalLibraryLinks
{
	class LibraryPage
	{
		public string Id { get; set; }
		public string Name { get; set; }
		public int Order { get; set; }

		public LibraryFolder[] Folders { get; set; }
	}
}

File: WebSalesLibraries/protected/models/services_data/common/dictionaries/LibraryPage.php
<?
	namespace application\models\services_data\common\dictionaries;

	/**
	 * Class LibraryPage
	 */
	class LibraryPage
	{
		public $id;
		public $name;
		public $order;

		/** @var  LibraryFolder[] */
		public $folders;
	}